app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -26,6 +26,15 @@
             {{#t}}Download Firefox for mobile devices.{{/t}}
           </p>
 
+          <form class="marketing-sms-link">
```

Comment:
I dig it. Are we going to show this to users we can reasonably assume to be in the US, maybe based off of locale? What happens if a non-US number is entered, will basket return an error message?